src/pages/documentation/platform-assurance/tutorials/push-debug-view.md

```diff
@@ -0,0 +1,119 @@
+# Push Debug View
+
+## Overview
+
+The Push Debug View provides the ability to validate the Push setup for your app and send a test message to your device.
+
+## Guide
+
+### Clients
+
+![Push Clients](./assets/push-debug-view/push-clients.png)
+
+The client dropdown has a list of each unique client that has connected to this Griffon session. A client is either a unique device or a unique app install for a device. For example, if an Android device and an iOS device have been connected to the session, those clients would appear in the Clients dropdown.
```

Comment:
I think it's no longer Griffon as well - shouldn't it be "... to this Platform Assurance session."?